doc/mobius/implementing-a-new-screen.md

```diff
@@ -265,7 +265,8 @@ interface UiActions {
 
 ### Wiring it all together
 
-We have implemented [`BaseScreen`](https://github.com/simpledotorg/simple-android/blob/master/doc/arch/011-screen-navigation-v2.md)
+We have
+implemented [`BaseScreen`](https://github.com/simpledotorg/simple-android/blob/master/app/src/main/java/org/simple/clinic/navigation/v2/fragments/BaseScreen.kt)
```

Comment:
Instead of this, can you add a permalink to this file, please? 

This way even the file location is changed or the file is renamed. We will still have the original link to the file.